### ISSUE TITLE
Image with src has a "loader" property that does not implement width. Please implement it or use the "unoptimized" property instead.

### ISSUE THREAD

**Issue by ElektrikSpark** [bundlesandbatches/next-sanity-image#50]
Seems that the loader property is not optimized.

[Next.js](https://nextjs.org/docs/messages/next-image-missing-loader-width)

**Comment by shaunchander** [comment]
Also experiencing this issue when using the `fill` property.

**Comment by lorenzodejong** [comment]
I've just released version `6.0.0`, could you check if the issue still occurs?

Also check out the updated installation docs (https://github.com/lorenzodejong/next-sanity-image#installation). Some people were running into an issue where specifying `loader: "custom"` in the `next.config.js` caused similar errors with the loader prop.

Otherwise please provide a reproducible case (from a CodeSandbox). I don't seem to be getting this error from my local setup.

**Comment by michael-land** [comment]
still see this issue with `6.0.0`

**Comment by mitchuman** [comment]
Seeing the issue for all of my images on `v6.0.0`. I'm not sure if my implementation is invalid or there is something wrong with `next-sanity-image`.

Here is how I've implemeted (simplified):
```tsx
// Img.tsx
import Image from 'next/image'

return <Image
  // @ts-ignore
  {...useNextSanityImage(client, image, options)}
  className={className}
  alt={alt || image.alt || ''}
  {...props}
/>
```
```tsx
// Component.tsx
import Img from 'Img'

return <Img
  image={image}
  options={{ imageBuilder: b => b.width(400) }}
/>
```

Which produces the following errors in the browser console logs:
```
Image with src "https://cdn.sanity.io/images/...-500x500.png?h=400&auto=format"
has a "loader" property that does not implement width.
Please implement it or use the "unoptimized" property instead.
Read more: https://nextjs.org/docs/messages/next-image-missing-loader-width
```

**Comment by philipppolder** [comment]
Any update on this?

**Comment by bradley** [comment]
Also seeing this.

**Comment by lorenzodejong** [comment]
As mentioned before, if you could provide a reproducible example in a Codesandbox (without your Sanity token) i could take a look at the issue. For now i can't reproduce the issue on my local environment, so it seems like this has something to do with specific setup and/or dependency versions.

**Comment by philipppolder** [comment]
Hi Lorenzo, here's a minimal CodeSandbox: 
https://codesandbox.io/p/sandbox/quizzical-heisenberg-fdwpfd

You just need to add your own projectId and any asset object. The warning should be in the browser's console.

Hope this helps and thanks for taking the time!

**Comment by lorenzodejong** [comment]
@mitchuman in your example you're actually circumventing the use of a responsive image. You're always returning an image with a width of 400, are you sure you require responsive images in your application?

**Comment by luigifgp** [comment]
this issue persist on next 13 using app directory

**Comment by andershagbard** [comment]
Persisting in Next14 app directory

**Comment by Ojay** [comment]
Yeah I'm seeing the same in Next 14, [codebox here.](https://codesandbox.io/p/devbox/winter-leaf-6n2s94?file=%2Flib%2Fclient.js)

Any help is greatly appreciated, thanks!

**Comment by Andrewryanhyde** [comment]
Also seeing this
